Assets/Scripts/Framework/Animation/AnimationController.cs

```diff
@@ -0,0 +1,44 @@
+using System.Linq;
+using UnityEngine;
+
+namespace Framework.Animation
+{
+    [RequireComponent(typeof(Animator))]
+    public sealed class AnimationController : MonoBehaviour
+    {
+        private const string INVALID_ANIMATION = "Animation name does not exist in the Animator controller: ";
+        private const string NO_ANIMATOR_ERROR = "Animator component is not assigned.";
+        
+        private Animator _animator;
+        private void Awake() => _animator = GetComponent<Animator>();
```

Comment:
```suggestion
        private Animator _animator;
        
        private void Awake() => _animator = GetComponent<Animator>();
```
[Hou die ene line.](https://github.com/Team-Swamp/IceBites/wiki/Code-convention#functions-1)